source/extensions/load_balancing_policies/maglev/maglev_lb.cc

```diff
@@ -13,6 +13,10 @@ bool shouldUseCompactTable(size_t num_hosts, uint64_t table_size) {
     return false;
   }
 
+#ifdef MAGLEV_LB_FORCE_ORIGINAL_IMPL
```

Comment:
Thinking out loud, will it make more sense to switch `MaglevTable::MaxNumberOfHostsForCompactMaglev` to data member, and initialize it with a different value for compact representation (and also for relevant tests)?
Alternatively, there could be an ifdef that assigns a different value to `MaglevTable::MaxNumberOfHostsForCompactMaglev`.

Reply:
Given that that value is more w.r.t a limit (also the tests mostly run with few number of hosts) I think that would be more convoluted.